src/openzaak/components/zaken/tests/test_cloud_events.py

```diff
@@ -1177,3 +1182,292 @@ def test_zaak_verlengen_post_sends_gemuteerd_cloud_event(self):
 
             self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
             self.assert_cloud_event_sent(ZAAK_GEMUTEERD, self.zaak, mock_send)
+
+
+class IncomingZaakCloudEventTests(JWTAuthMixin, APITestCase):
+    heeft_alle_autorisaties = True
```

Comment:
To make sure it's tested that this specific scope gives permission to publish events
```suggestion
    heeft_alle_autorisaties = False
```